docs/conf.py

```diff
@@ -1,13 +1,30 @@
 # -*- coding: utf-8 -*-
 
-import django
 import os
 
-base_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)), "..")
+import django
+from django.conf import settings
 
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'tests.settings')
+base_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 # Manual setup is required for standalone Django usage
+# NOTE: Since building documentation is independent from the tests, it can't
+# use the 'test.settings' Django settings module.
```

Comment:
`test.settings` can be used to build documentation, but are not included in the build. Please update note to reflect that.